lib/modules/neovim.nix

```diff
@@ -0,0 +1,30 @@
+{ lib }:
+let
+  inherit (lib)
+    mkEnableOption
+    mkOption
+    types
+    ;
+
+in
+{
+  # TODO: Bring this from home-manager!
```

Comment:
Is there a PR for this? Can we already use it?

Reply:
There's no PR yet. The scope was too large already I wanted to get these merged before we add/refactor even more.

Reply:
Let's table that for later then.